contributing/development-setup.md

```diff
@@ -51,7 +51,20 @@ For more explained information about plugin override check [Development Override
 - Make sure that the PR title follows [*Conventional Commits*](https://www.conventionalcommits.org/).
 - Add comments around your new code that explain what's happening.
 - Commit and push your changes to your branch then submit a pull request against the `master` branch.
-- A repo maintainer will review the your pull request, and may either request additional changes or merge the pull request.
+- A repo maintainer will review the your pull request. 
+
+### Merging a Pull Request
+Due to security reasons, there are restrictions on how external contributions can be handled, especially concerning the use of repository secrets and running tests from forks.
+Our GitHub setup does not allow sharing of variables and secrets across forks made by external contributors. This is a necessary security measure to prevent unauthorized access and potential vulnerabilities
+
+As external contributors cannot merge changes from their forks directly nor create branches from the master, the contribution process involves co-authoring. Here's what will happen:
+
+- We will cherry-pick your commit into a new branch in our main repository.
+- We will then create a new pull request from this branch, adding you as a co-author to ensure your contribution is not lost.
```

Comment:
just for our clarity the co-author aspect is retained because of the cherry-pick, not specifically when opening the PR.

Reply:
yes